csharp/HOLMS.Platform/HOLMS.Platform/Support/Security/JWToken.cs

```diff
@@ -10,26 +10,28 @@ public sealed class JWToken {
         public const int AccessTokenExpirationMinutes = 60;
         public const string ClientIdKey = "client_id";
         public const string TenancyIdKey = "tenancy_id";
+        public const string UserEntitlementsKey = "user_entitlements";
         public const string UserIdKey = JwtRegisteredClaimNames.Sub;
         public const string Audience = "private.shortbar.com";
         public const string Issuer = "private.shortbar.com";
 
         public string SignedToken { get; private set; }
         public Guid Id { get; }
 
-        public JWToken(ClientInstanceIndicator client, StaffMemberIndicator user,
+        public JWToken(ClientInstanceIndicator client, StaffMemberIndicator user, dynamic userEntitlements,
```

Comment:
You should enforce more structure here since this is a public member.

I would make it a `List<Entitlement>` and put the "wire format codec" (the thing responsible for COding and DECoding (actual etymology there)) as a private member in HOLMS.PlatformClient.